src/styles/judging.scss

```diff
@@ -0,0 +1,191 @@
+.judging-who-are-you {
+  background: #1d8a4f;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  height: 100vh;
+}
+
+.who-are-you-title {
+  text-align: center;
+  font-weight: 900;
+  font-size: 64px;
+}
+
+.who-are-you-button {
+  width: 300px;
+  margin: 30px auto;
+  background: $dark-blue;
+  color: white;
+  font-size: 24px;
+}
+
+.judging-find-person {
+  background: #1d8a4f;
+  display: flex;
+  justify-content: center;
```

Comment:
probably would be good to toss in an align-items: center here (i think i mentioned this previously too). also maybe could get rid of justify-content center and add in some margin top instead? kinda pushes down the dropdown a bit and i dont think it has to

Reply:
were you saying that I should just use all the styling from their example? I initially did that but then removed some because I didn't think it was actually doing anything. I will add it back.

Reply:
wait @bugsalexander do you wanna help me out a ton by fixing the styling? I will def fix the other stuff you commented tho, appreciate the review!